packages/app-project/package.json

```diff
@@ -52,7 +52,7 @@
     "newrelic": "~7.1.0",
     "next": "~10.1.3",
     "node-fetch": "~2.6.1",
-    "panoptes-client": "~3.2.1",
+    "panoptes-client": "3.3.2",
```

Comment:
```suggestion
    "panoptes-client": "~3.3.2",
```

Reply:
i noticed this - i found it a little strange that `yarn upgrade` didn't respect the ~ already there.

Reply:
Using `yarn upgrade <package>@<version>` will upgrade the package to the exact version you specify (which is different behavior from npm). Use `yarn upgrade <package> --latest` for it to follow the existing range specifier. I prefer using `yarn upgrade-interactive --latest`.